### ISSUE TITLE
Fix Localization issue with long key

### ISSUE THREAD

**Issue by AliSoftware** [wordpress-mobile/WordPress-iOS#15353]
TL;DR: One of the L10n keys we use is too long and causes issues with GlotPress. We need to split it in 2

### Context

I'm currently implementing an automated linting of our `.strings` file to detect any inconsistency between the translation of some of our keys across different locales. [Ref: paaHJt-D6-p2]

So far my tool has found existing inconsistencies:

 - Some are because of translators not using the right `%d/%@/…` placeholders in their translations; for those I will fix this directly in GlotPress.
 - But one of the issue is with one of our localization key whose length exceeds the 255 limit that GlotPress seems to have on key names. That's the focus of this GH issue

### The Bug

The key in question is used in `WordPress/Classes/ViewRelated/Blog/Site Management/StartOverViewController.swift`

* The full (long!) key is: `"If you want a site but don't want any of the posts and pages you have now, our support team can delete your posts, pages, media, and comments for you.\n\nThis will keep your site and URL active, but give you a fresh start on your content creation. Just contact us to have your current content cleared out."`
* But in practice it gets truncated during the localisation process, right after `Just cont` (256 char limit).

### Expected behavior

The text for this copy should correctly appear translated in the app

### Actual behavior

Because the key name gets truncated during the round trip to GlotPress, when iOS tries to find a translation from the full long key at runtime, it won't find a matching translation in the `Localizable.strings` if your language is not `en` (the only `.strings` file that has the key untruncated). So instead it will fallback to using the key, thus showing the English copy to the user instead of the translated copy.

### Steps to reproduce the behavior

* Compare line 2822 of `en.lproj/Localizable.strings` with the same from any other `*.lproj/Localizable.strings`
* You could also test it at runtime by setting your phone to a locale other than english and navigate to the Start Over screen to see that text appear in English despite the phone being in another language.

##### Suggested Fix

To work around the 255 character limit, given that this copy seems to basically be 2 separate paragraphs after all, I suggest we split that text in 2 separate keys, one for each paragraph.

_Once the 2 new keys have been created and used in the code – and the old key is removed from the code – one should ideally go to GlotPress to do the same split of the 2 paragraphs in all the translations for that text, so that we keep using the same translation for those paragraphs than before without having our community to re-translate the newly-introduced key_
